Makefile

```diff
@@ -47,3 +47,5 @@ $(EXTENSION)--2.15.sql: $(EXTENSION)--2.14.sql $(EXTENSION)--2.14--2.15.sql
 	cat $^ > $@
 $(EXTENSION)--2.16.sql: $(EXTENSION)--2.15.sql $(EXTENSION)--2.15--2.16.sql
 	cat $^ > $@
+$(EXTENSION)--2.17.sql: $(EXTENSION)--2.16.sql $(EXTENSION)--2.16--2.17.sql
+	cat $^ > $@
```

Comment:
We no longer need to generate these files, as PG >= 10 is clever enough to find migration paths automatically.

No need to change anything here, I'll open an issue to track and fix this later.